### ISSUE TITLE
Http Json Api Server: verify wrong? and unixtime

### ISSUE THREAD

**Issue by spottuit** [rvagg/learnyounode#278]
So my first problem is that I somehow have a difference of two hours between my outcome of the time and the outcome from the learnyounode verify command:
![d73c28c5b21f4a7a559075ca4ca2dc6b](https://cloud.githubusercontent.com/assets/9430751/7258152/fb778ede-e85a-11e4-8d14-1ce68ddc188a.png)

Maybe it is important to say that 18 was the time on my computer at the time I made the screenshot.  Anyway when I used the learnyounode run command I saw that my time matched the time from the query:
![3ff811f1d6c404ae5bfb3212864a69ea](https://cloud.githubusercontent.com/assets/9430751/7258290/dc2bcae4-e85b-11e4-8233-07c0c0a96e5b.png)

My second problem is that my unix timestamp isn't right, but I can't figure out why. Are there any extra things like leap years? Or do I have a  fault in my code?

Code from main file:
![c288f5247aea2fd3d925a81579deddde](https://cloud.githubusercontent.com/assets/9430751/7258680/60e78ffa-e85e-11e4-96f5-2875b4167951.png)

Code from the unixtime-module:
![44f43c342140a92f8abe833ac640d1cd](https://cloud.githubusercontent.com/assets/9430751/7258696/83393e64-e85e-11e4-93d4-226ad9c3f6d4.png)

PS: I know I can get the unix timestamp in a shorter way, but I want to learn more and challenge myself.

**Comment by martinheidegger** [comment]
Did you check the time zones? GMT time versus local time?

**Comment by spottuit** [comment]
I see that the GMT time is now: 18:09 and my time is 20:09. So I guess I have to add 2 hours extra to the total of miliseconds and also add two hours to the first JSON response. But then we have still the second problem left.

**Comment by martinheidegger** [comment]
Did you know that you can pass hour, day, month etc. to the date object? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date

Also: it is possible to add formatted code to github: https://help.github.com/articles/github-flavored-markdown/#syntax-highlighting

**Comment by spottuit** [comment]
Yes I know I can use the date object to get the unix timestamp, but I wanted to do it without the date object.
